civiclens/templates/document.html

```diff
@@ -8,7 +8,7 @@
 {% block content %}
 
 <!-- whole page -->
-<section class="slice py-3" style="background-color:#FDFFF7" id="core-block">
+<section class="slice py-3 pb-6" style="background-color:#FDFFF7" id="core-block">
```

Comment:
`<section class="slice py-3 pb-10" style="background-color:#FDFFF7" id="core-block">`

I checked this on the code I made a PR on and 6 was an improvement, but still had issues, on 5 that I quickly checked changing this to 10 seemed to work

Reply:
yeah it's super weird- this is definitely not solving the root cause of the issue, but could be enough to get us through tomorrow.

Do you think i should spend a bit more time looking into it or stick with pb-10 for now?

Reply:
We can stick to pb 10 for now and reflag it if it's a noticeable issue when the backend bros start testing everything out